### PR TITLE
New package: geth-1.8.17

### PR DESCRIPTION
**Pull request by sihoang**

### CODE REVIEW

File: srcpkgs/geth/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+wrksrc="go-ethereum-${version}"
+build_wrksrc="go-ethereum-${version}"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/ethereum/go-ethereum"
+go_package="${go_import_path}/cmd/geth"
+go_build_tags="v${version}"
+short_desc="Official Go implementation of the Ethereum protocol"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="LGPL-3.0"
```
Comment (maxice8) [inline srcpkgs/geth/template]: please use SPDX identifier

File: srcpkgs/geth/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'geth'
+pkgname=geth
+version=1.8.17
+revision=1
+wrksrc="go-ethereum-${version}"
+build_wrksrc="go-ethereum-${version}"
```
Comment (maxice8) [inline srcpkgs/geth/template]: no need for build_wrksrc= neither create_wrksrc=

File: srcpkgs/geth/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/ethereum/go-ethereum"
+go_package="${go_import_path}/cmd/geth"
+go_build_tags="v${version}"
+short_desc="Official Go implementation of the Ethereum protocol"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="LGPL-3.0"
+homepage="https://geth.ethereum.org"
+distfiles="https://${go_import_path}/archive/${go_build_tags}.tar.gz"
```
Comment (maxice8) [inline srcpkgs/geth/template]: please use v${version}

**Comment by sihoang** [comment]
@maxice8 thanks for reviewing.  Please excuse my stupid mistake as this is my first time using xbps.  I've just installed void-linux over the weekend and am excited to be part of the community.

**Pull request merged**